apps/package.json

```diff
@@ -24,7 +24,7 @@
     "test-audio": "echo \"Open your browser to http://127.0.0.1:8080/test/audio/audio_test.html\" && http-server .",
     "build:log": "grunt logBuildTimes",
     "build:stats": "node ./script/build-stats.js",
-    "prestorybook": "curl -o build/package/css/application.css http://localhost-studio.code.org:3000/assets/application.css",
```

Comment:
This url should still work after the upgrade (though obviously it stopped working for a bunch of us)... maybe we should revert this fix after the domain serving is fixed?